readthedocs/builds/migrations/0007_add-automation-rules.py

```diff
@@ -32,7 +32,7 @@ class Migration(migrations.Migration):
             ],
             options={
                 'ordering': ('priority', '-modified', '-created'),
-                'manager_inheritance_from_future': True,
+                # 'manager_inheritance_from_future': True,
```

Comment:
I'm not 100% sure if we need to do something here to support this: https://docs.djangoproject.com/en/3.2/releases/1.10/#manager-use-for-related-fields-and-inheritance-changes

This option comes from django-polymorphic. cc @stsewd

Reply:
Looks like this was to opt-in into the behavior from 3.x, so should be fine.